### PR TITLE
feat: allow redefine tag separator

### PR DESCRIPTION
**Pull request by dohnto**
@freznicek @lksv

### CODE REVIEW

File: generate_dockerfiles.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def convert_docker_tag_to_makefile_target(docker_tag):
     return docker_tag.replace('/', '__').replace(':', '~')
 
 
-def generate_dockerfiles(output_dir, configuration_files):
+def generate_dockerfiles(output_dir, configuration_files, tag_separator):
```
Comment (freznicek) [inline generate_dockerfiles.py]: You may want to use tag_separator='_' parameter default. (backward compatibility at function level)

**Comment by freznicek** [comment]
LGTM

**Comment by lksv** [comment]
There is a section in README which should be updated.

File: generate_dockerfiles.py (hunk 1 above)
Comment (dohnto) [reply generate_dockerfiles.py]: addressed in: 112abf0

**Comment by dohnto** [comment]
@lksv addressed in: 8d94d4b

**Pull request merged**